File: api-java-ecommerce/src/main/java/busness/ecommerce/entity/Setting.java
package busness.ecommerce.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Data
@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Setting {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String nameApp;

    @Column(nullable = false)
    private String nameDev;

    @Column(nullable = false)
    private String version;

    @Column(nullable = false)
    private String theme;

    @Column(nullable = false)
    private String textColor;

    @Column(nullable = false)
    private String currency;

    @Column(nullable = false)
    private String address;

    @Column(nullable = false)
    private String email;

    @Column(nullable = false)
    private String phone;

    @Column(nullable = false)
    private String logo;

    @Column(nullable = false)
    private Integer width;

    @Column(nullable = false)
    private Integer height;
}

File: lisungi/src/app/core/models/Setting.ts
export interface Setting{
  publicId?: number;
  nameApp: string;
  nameDev: string;
  version: string;
  theme: string|null;
  bodyTheme: string|null;
  textColor: string|null;
  currency: string;
  address: string;
  email: string;
  phone: string;
  logo: string;
  width: number;
  height: number;
}
